server/config/client.config.ts

```diff
@@ -81,27 +81,36 @@ export function generateConfig(params: ConfigGeneratorParams, env: Env) {
       MOBILE_BASE: env.URL_MOBILE_BASE,
       PRICING: env.URL_PRICING,
       PRIVACY_POLICY: env.URL_PRIVACY_POLICY,
+      SUBPATH: {
```

Comment:
I think `SUBPATH` is not the right wording for it. Maybe `URL_PATH` would do? WDYT?

Reply:
Wasn't too sure about subpath tbh.
That would require updating the default config, and since we're at it, maybe we could get rid of the granularity and use a full path for all 4?
The granularity is not taken advantage of in the app and that sounds more straightforward to me for end user config, WD*Y*T?

Reply:
yes, let's flatten those URL, no more `BASE` URLs, I'd be up for that